model/src/main/java/com/basistech/rosette/apimodel/recordsimilarity/RecordSimilarityFieldInfo.java

```diff
@@ -29,7 +30,7 @@
 @Value
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class RecordSimilarityFieldInfo {
-    @NotNull @Valid String type;
+    @Getter @NotNull @Valid String type;
```

Comment:
why is this getter necessary?  don't we already get that from the `@Value` annotation?

Reply:
I added this because I had builds that were looking for a `getType` method, but weren't able to find it. Was this something we had removed as part of earlier changes? If so, meaning I only added it back because of my efforts to simplify the PR, I can remove it.

Reply:
every field in this class should have a getter, but [according to lombok](https://projectlombok.org/features/Value) the `@Value` annotation is shorthand for a bunch of different annotations, one of which is `@Getter`.  Maybe you had a weird local state?  Can you reproduce the error with this branch if you remove the getter annotation?

Reply:
I'm trying to replicate this now, not sure if it will work after making some changes to resolve merge conflicts. But definitely weird

Reply:
I'm waiting for a full SDK build, just in case I had built something different (I don't think I have), but it seems I'm not getting this error anymore? Not sure what happened here. If I still don't get the error after building the SDK, I'll remove this.

Reply:
That took me a little longer than expected, but I am no longer getting this error after removing the getter annotation locally again. I'm not sure what caused this to be happen, but I'll remove it from the PR.